docs/ocp4-manual-deployment-dhcp.md

```diff
@@ -108,7 +108,7 @@ networking:
   - cidr: 10.254.0.0/16
     hostPrefix: 24
   networkType: OpenShiftSDN
-  serviceNetwork:
+  service_network:
```

Comment:
This should remain as `serviceNetwork` here as per my understanding.

Reply:
yes, totaly correct